bp-esa-frontend/src/components/filterSearchForm.js

```diff
@@ -1,49 +1,61 @@
-                                        type="radio"
-                                        className="form-check-input"
-                                        name="filter"
-                                        id={criteria.name}
-                                        value={criteria.name}
-                                        />
-                                        {criteria.display}
-                                    </label>
-                                    </div>
-                                 ))
-                            }
-                        </fieldset>
-                        </Col>
-                        <Col sm={9}>
-                        <div className="form-group">
-                            <input
-                            type="search"
-                            className="form-control"
-                            placeholder="&#xF002; Search"
-                            id="search"
-                            name="search"
-                            />
-                        </div>
-                        </Col>
-                    </Row>
-                </form>
-            </div>
-        );
-    }
+const FilterSearchForm = (props) => {
+    const { filterCriteria, path } = props;
+    return(
+        <div>
+            <form id="filterForm">
+                <Row>
+                    <Col sm={3}>
+                    <fieldset className="form-group">
+                        {
+                                filterCriteria.map(criteria => (
+                                <div className="form-check">
+                                <label className="form-check-label">
+                                    <input
+                                    type="radio"
+                                    className="form-check-input"
+                                    name="filter"
+                                    id={criteria.name}
+                                    value={criteria.name}
+                                    />
+                                    {criteria.display}
+                                </label>
+                                </div>
+                                ))
+                        }
+                    </fieldset>
+                    </Col>
+                    <Col sm={9}>
+                    {
+                        (path === '/reports') &&
+                            <div className="form-group">
+                                <input
+                                type="search"
+                                className="form-control"
+                                placeholder="&#xF002; Search"
+                                id="search"
+                                name="search"
+                                />
+                            </div>
+                    }
+                    </Col>
+                </Row>
+                {
+                    (path === '/') &&
```

Comment:
Same here, should this be checking if the current path is `/` then displaying the status drop down?

Reply:
Yes to this too